### ISSUE TITLE
Loss in run_gen.py

### ISSUE THREAD

**Issue by blurLake** [salesforce/CodeT5#38]
Hi, I am trying to figure out how the loss is calculated, for example [here](https://github.com/salesforce/CodeT5/blob/34a6310f9604f77031b6a412a1b2b3873dbd6e19/run_gen.py#L236). I assume it is some distance between generated_ids and target_ids with attention_masks, but could you point it out how the code and formula look like? Thank you very much!

**Comment by yuewang-cuhk** [comment]
HI, as CodeT5 is adapted from Huggingface's T5 implementation, you should refer to official documentations like [here](https://huggingface.co/docs/transformers/model_doc/t5#transformers.T5ForConditionalGeneration).
